MSAL/src/public/MSALPublicClientApplication.h

```diff
@@ -34,50 +34,156 @@
 
 @interface MSALPublicClientApplication : NSObject
 
+/*! Whether or not to validate the authority used with the application, default YES */
 @property BOOL validateAuthority;
+
+/*! The authority used with the application object */
 @property (readonly) NSURL *authority;
```

Comment:
great opportunity to briefly explain what an authority is. :) We don't want to basically copy Vittorio's book into this file, but something brief may be helpful.